### PR TITLE
feature/EB-3004550541: Portkey V2 Integration

### PR DESCRIPTION
**Pull request by faizalebridge**
Portkey V2 Integration

### CODE REVIEW

File: rewrites.test1.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
-const host = 'http://192.168.66.78:7030';
+/* const host = 'http://192.168.66.78:7030';
 module.exports = [
   { source: '/api/:path*', destination: `${host}/api/:path*` },
   { source: '/cms/:path*', destination: `https://cms.ebridge.exchange/:path*` },
   { source: '/AElfIndexer_eBridge/:path*', destination: `http://192.168.66.78:7022/AElfIndexer_eBridge/:path*` },
+];*/
+const host = 'https://test.ebridge.exchange';
```
Comment (yongebridge) [inline rewrites.test1.js:7]: @faizalebridge is this correct?

File: src/constants/test1/AELF.ts [hunk 2]
```diff
@@ -1,12 +1,14 @@
 export const CHAIN_INFO = {
   chainId: 'AELF',
   exploreUrl: 'https://explorer-test.aelf.io/',
-  rpcUrl: 'http://192.168.66.61:8000',
+  rpcUrl: 'https://aelf-test-node.aelf.io',
```
Comment (yongebridge) [inline src/constants/test1/AELF.ts:4]: Is this correct? Seems to be testnet and not test1

**Review by yongebridge — CHANGES_REQUESTED** [review]

File: rewrites.test1.js (hunk 1 above)
Comment (faizalebridge) [reply rewrites.test1.js]: Updated

File: src/constants/test1/AELF.ts (hunk 2 above)
Comment (yongebridge) [reply src/constants/test1/AELF.ts:4]: is this file changed?

Comment (faizalebridge) [reply src/constants/test1/AELF.ts]: Please check now.

**Review by yongebridge — APPROVED** [review]

**Pull request merged**